package.json

```
@@ -812,7 +812,7 @@
         "command": "mdb.saveMongoDBDocument",
         "key": "ctrl+s",
         "mac": "cmd+s",
-        "when": "mdb.isPlayground == true"
+        "when": "editorLangId == json"
```

Will this run on all json files? Or is this hot key only used when we have the extension open?

---

Maybe not for this pr, unless you want to, would it make sense to not handle every save event ourselves there?
Instead we could update this `when` clause to check for `resourceScheme` which indicates a mongodb file. 
https://code.visualstudio.com/api/references/when-clause-contexts
```suggestion
        "when": "resourceScheme == VIEW_DOCUMENT_SCHEME"
```
Scheme from https://github.com/mongodb-js/vscode/blob/main/src/editors/mongoDBDocumentService.ts#L20

It's a bit obscure, but I'm thinking if another extension is used for formatting json or doing something when json is saved this current approach we have would conflict with that. There's one other place where we use the `editorLangId == json` `when` clause which we could update as well.